ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -866,33 +886,41 @@ Model Response Should Match The Expectation
         Should Be Equal    ${cleaned_response_text}    ${cleaned_exp_response_text}
     END
 
-Query Models And Check Responses Multiple Times
+Query Model Multiple Times
     [Documentation]    Queries and checks the responses of the given models in a loop
     ...                running ${n_times}. For each loop run it queries all the model in sequence
-    [Arguments]    ${models_names}    ${namespace}    ${endpoint}=${CAIKIT_ALLTOKENS_ENDPOINT}    ${n_times}=10
-    ...            ${streamed_response}=${FALSE}    ${query_idx}=0
+    [Arguments]    ${model_name}    ${namespace}    ${isvc_name}=${model_name}    ${endpoint}=${CAIKIT_ALLTOKENS_ENDPOINT}    ${n_times}=10
+    ...            ${streamed_response}=${FALSE}    ${query_idx}=0    ${validate_response}=${TRUE}    &{args}
+    IF    ${validate_response} == ${FALSE}
```

Comment:
Do you think it makes sense to follow Robocop advice and simplify this to:
```suggestion
    IF    ${validate_response}
```

(the true/false code block branches need to be switch in such case of course)

Reply:
it doesn't open the details of the alert, however it was originally done with IF var == TRUE, but it doesn't work for all the combinations we may have, because in the ELSE statement I'm not setting the var to a static value, but to another variable.

Reply:
besides `IF ${validate_response}` is verified when the variable is not None